stats_server/Cargo.toml

```diff
@@ -4,8 +4,8 @@ version = "0.1.0"
 authors = ["kingoflolz <wangben3@gmail.com>"]
 
 [dependencies]
-actix = {git="https://github.com/kingoflolz/actix", branch="mocking"}
-actix-web = {git="https://github.com/kingoflolz/actix-web", default-features = false}
+actix = "0.5.7"
+actix-web = {version = "0.5.7" , default-features = false}
```

Comment:
actix web doesn't follow the same version as actix, so the latest version of actix web is actually 0.6.7